mailbox/store/src/test/java/org/apache/james/mailbox/store/search/AbstractMessageSearchIndexTest.java

```diff
@@ -315,6 +315,30 @@ public void searchingMessageInMultipleMailboxShouldNotReturnLessMessageThanLimit
             .hasSize(limit);
     }
 
+
+    @Test
+    public void countMessageInMailboxesShouldReturnTotalNumberOfMessage() throws Exception {
+        Assume.assumeTrue(messageIdManager != null);
```

Comment:
How is **total** feature related with **messageId** one?

Can't we write a test that don't need that assumption?